atomic_reactor/pulp_util.py

```diff
@@ -183,8 +183,16 @@ def create_dockpulp_and_repos(self, image_names, repo_prefix="redhat-"):
     def get_image_ids_existing(self, layers):
         return self.p.getImageIdsExist(layers)
 
-    def upload(self, filename):
-        self.p.upload(filename)
+    def upload(self, filename, repo_id):
+        # returns boolean that is was uploaded to rh-everything
+        try:
+            self.p.upload(filename, repo_id)
+            self.log.debug("to the repo %s", repo_id)
```

Comment:
Do we want a more descriptive message here? :-)

Reply:
in caller there is already "uploading filename", but I can move it here?

Reply:
moving message to upload method